composer.json

```diff
@@ -19,10 +19,10 @@
     "require": {
         "php": "^5.6 || ^7.0",
         "simple-bus/message-bus": "~3.0",
-        "symfony/http-kernel": "~2.3 || ~3.0",
-        "symfony/dependency-injection": "~2.3 || ~3.0",
-        "symfony/yaml": "~2.3 || ~3.0",
-        "symfony/config": "~2.3 || ~3.0"
+        "symfony/http-kernel": "~2.7 || ~3.0 || ~4.0",
```

Comment:
If you take the chance to bump Symfony 2.x requirements to 2.7, I would also drop support for 3.0, 3.1, and 3.2.